hq/app/schedule/Dynamo.scala

```diff
@@ -73,7 +74,8 @@ class Dynamo(client: AmazonDynamoDB, tableName: Option[String]) extends Attribut
   def alertToMap(e: IamAuditAlert): Map[String, AttributeValue] = {
     Map(
       "date" -> N(e.dateNotificationSent.getMillis),
-      "notificationType" -> S(e.notificationType.name)
+      "disableDeadline" -> N(e.disableDeadline.getMillis)
+      //"notificationType" -> S(e.notificationType.name)
```

Comment:
should these lines still be removed?

Reply:
Great spot, thanks Phil! I've removed these commented out lines now.